devstack.sh

```diff
@@ -1,5 +1,6 @@
 #! /bin/sh
 useradd -s /bin/bash -d /opt/stack -m stack
+chmod 755 /opt/stack
 echo "stack ALL=(ALL) NOPASSWD: ALL" | tee /etc/sudoers.d/stack
 su - stack
 yum -y install git
```

Comment:
vipin3699, This step will require sudo privilege for yum to install git.

Reply:
will change that asap